styles.css

```diff
@@ -1,13 +1,69 @@
-.scryfall_hover-zoom { transition: transform .2s; width: 200px; position: relative; z-index: 10; }
-.scryfall_hover-zoom:hover { transform: scale(1.5); z-index: 100; }
-.scryfall_hover-zoom--container { position: relative; }

```

Comment:
i would prefer to not reformat these to keep the diff focussed on the new features.

Reply:
i will change it back then ^^ probably won't get to work on it before 🇷🇺 though... but i will need to find out how to implement my hook first... is there a way i can work directly on a live demo ? (so i can play around with the php as well)